package.json

```diff
@@ -1,29 +1,36 @@
 {
-	"name": "serialize-error",
+	"name": "@ponbike/serialize-error",
 	"version": "11.0.2",
 	"description": "Serialize/deserialize an error into a plain object",
 	"license": "MIT",
-	"repository": "sindresorhus/serialize-error",
+	"repository": "ponbike/serialize-error",
 	"funding": "https://github.com/sponsors/sindresorhus",
```

Comment:
Funding?

Reply:
It's the author of  serialize-error so I keep it in